aws/table_aws_codedeploy_deployment_config.go

```diff
@@ -0,0 +1,203 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/codedeploy"
+	"github.com/aws/aws-sdk-go-v2/service/codedeploy/types"
+
+	codedeployv1 "github.com/aws/aws-sdk-go/service/codedeploy"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCodeDeployDeploymentConfig(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_codedeploy_deployment_config",
+		Description: "AWS CodeDeploy Deployment Config",
+		Get: &plugin.GetConfig{
+			KeyColumns:   plugin.SingleColumn("deployment_config_name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"DeploymentConfigDoesNotExistException"}),
+			},
+			Hydrate: getCodeDeployDeploymentConfig,
+		},
+		List: &plugin.ListConfig{
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"DeploymentConfigDoesNotExistException"}),
+			},
+			Hydrate: listCodeDeployDeploymentConfigs,
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(codedeployv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) specifying the application.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getCodeDeployDeploymentConfigArn,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "compute_platform",
+				Description: "The destination platform type for the deployment (Lambda, Server, or ECS).",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getCodeDeployDeploymentConfig,
+			},
+			{
+				Name:        "create_time",
```

Comment:
@madhushreeray30 this is resulting <null> all the time? What is AWS CLI response?